spec/PeriodicBackgroundSync-index.bs

```diff
@@ -0,0 +1,95 @@
+Repository: WICG/BackgroundSync
+Link Defaults: html (dfn) allowed to show a popup/event handler idl attribute/global object/in parallel/incumbent settings object/perform a microtask checkpoint/queue a task/script execution environment
+</pre>
+
+<pre class="anchors">
+spec: html; urlPrefix: https://html.spec.whatwg.org/
+    type: dfn
+        text: trusted; url: concept-events-trusted
+        text: browsing context; url: browsing-context
+
+spec: service-workers; urlPrefix: https://slightlyoff.github.io/ServiceWorker/spec/service_worker/index.html
+    type: dfn
+        text: active worker; url: dfn-active-worker
+        text: client; url: dfn-service-worker-client
+        text: control; url: dfn-control
+        text: extended lifetime promises; url: dfn-extend-lifetime-promises
+        text: frame type; url: dfn-service-worker-client-frame-type
+        text: handle functional event; url: handle-functional-event-algorithm
+        text: installing worker; url: dfn-installing-worker
+        text: service worker; url: service-worker-concept
+        text: service worker client; url: dfn-service-worker-client
+        text: service worker registration; url: service-worker-registration-concept
+        text: termination; url: terminate-service-worker-algorithm
+        text: waiting worker; url: dfn-waiting-worker
+    type: interface
+        text: ExtendableEvent; url: extendable-event-interface
+        text: ExtendableEventInit; url: extendable-event-init-dictionary
+        text: ServiceWorkerGlobalScope; url: service-worker-global-scope-interface
+        text: ServiceWorkerRegistration; url: service-worker-registration-interface
+</pre>
+
+Introduction {#intro}
+=====================
+
+  <em>This section is non-normative.</em>
+
+  Web Applications often run in environments with unreliable networks (e.g., mobile phones) and unknown lifetimes (the browser might be killed or the user might navigate away). This makes it difficult to synchronize client data from web apps (such as photo uploads, document changes, or composed emails) with servers. It also makes it difficult to synchronize updated content on the servers with the web app. If the browser closes or the user navigates away before synchronization can complete, the app must wait until the user revisits the page to try again. This specification provides a new onperiodicsync [=service worker=] event which can fire [=in the background=] so that synchronization attempts can continue despite adverse conditions when initially requested. This API is intended to reduce the time between content creation and content synchronization between the server and the web app.
+
+  As this API relies on service workers, functionality provided by this API is only available in a [=secure context=].
+  ## Example ## {#example}
+  Requesting a [=periodic Background Sync opportunity=] at a daily frequency from a [=browsing context=]:
+
+  <pre class="lang-js">
+    async function getLatestNews() {
```

Comment:
nit: `getLatestNews` -> `registerPeriodicNewsCheck` or similar, as it doesn't actually get the latest news.